Accessibility check and report

Sirvelasque opened the pull request:
- Page title - no issues found
- Image text alternative - no issues found
- Text heading - no issues found
- Color contrast - [buttons text color](https://github.com/Sirvelasque/portfolio/pull/10/files#diff-1fc556f95754ee7e33d91044125c44bb9f750c99be4406756ffb27413adfcaf5L447-R447) had 3.1 contrast with background color
- Resize - no issues found. Fixed
- Interaction - no issues found
- Moving content - no issues found
- Multimedia - no issues found
- The basic structure of the page - [Form inputs](https://github.com/Sirvelasque/portfolio/pull/10/files#diff-0eb547304658805aad788d320f10bf1f292797b5e6d745a3bf617584da017051R176-R191) had no labels. Fixed

Sirvelasque committed:
disable images for structures check

Have-Samuel committed:
Restoring images in index and CSS

Sirvelasque committed:
restored suptext color

Deepakdanger reviewed (changes requested):
Hi @Sirvelasque and @Have-Samuel, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️

Since you have not changed the file `index.html`, I can't able to point specifically, that's why I'm requesting changes here only. Please check and fix the issue.

- [ ] Every form control has a label associated with it using 'label', 'for', and 'id'. All the inputs under the form line `173` in `index.html` we don't have label for now. Please add it.

- [ ] The page has a minimum contrast ratio of at least 4.5:1 for normal-size text. But there are all the button `See Project` has low contrast. It is on line `112`, `127`, `142`, `157` in the `index.html`. You can use [this aditus tool](https://www.aditus.io/button-contrast-checker/) or [wave tool extension](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh) to help checking the color contrast of the project. Please fix it.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Sirvelasque committed:
add labels to contact form items

for accessibility now it has labels that can be seen if stylesheet doesn't work properly

Have-Samuel committed:
Changed the font-color of the buttons

Have-Samuel committed:
Merge branch 'Accessibility' of github.com:Sirvelasque/portfolio into Accessibility

Sirvelasque committed:
fix linter errors

- duplicate ID
- charset not properly setted
- color setted with upper cases on hex code

ReshyResh reviewed (approved):
# Hi team, 

### Good job addressing the changes requested by the previous reviewer.
#### Your project is complete!  There is nothing else to say other than... it's time to merge it!
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

----

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_
